torch/testing/_internal/common_device_type.py

```diff
@@ -232,6 +242,11 @@ def _get_precision_override(self, test, dtype):
             return self.precision
         return test.precision_overrides.get(dtype, self.precision)
 
+    def _get_tolerance_override(self, test, dtype):
+        if not hasattr(test, 'tolerance_overrides'):
+            return self.precision, self.rel_tol
+        return test.tolerance_overrides.get(dtype, (self.precision, self.rel_tol))
```

Comment:
Should this line be

```
return test.tolerance_overrides.get(dtype, tol(self.precision, self.rel_tol))
```

?

Reply:
Ok, this is more consistent for the get() function.